Lesson6/src/com/home/military/MilitaryOffice.java

```diff
@@ -71,4 +71,13 @@ private boolean isPersonFit(Person person) {
                 && person.getAge() > 17
                 && person.getAge() < 28;
     }
+
+    public void getMilitaryInfo() {
+        for (MilitaryUnit unit : units) {
+            System.out.println("Military unit number " + unit.getUnitNumber());
+            if (unit.getFreePlacesFromList() != 0) {
+                unit.infoRecrutsMilitaryUnit(unit);
```

Comment:
you can refactor this method to not have a military unit parameter because you are calling it from the military unit object